src/styles/blocks/star.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  align-items: center;
+
+  &__star {
+    background-image: url"../images/star.svg";
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &__star:nth-child(-n+4) {
```

Comment:
reuse logic from a previous `stars` task to be able to work with different amount of stars